pakyow-presenter/lib/pakyow/presenter/form.rb

```diff
@@ -0,0 +1,65 @@
+module Pakyow
+  module Presenter
+    class Form
+
+      attr_reader :form_view, :context
+
+      def initialize(view, context)
+        @form_view = view
+        @context = context
+      end
+
+      def create(*binding_args, &block)
+        bind_data(binding_args, block) do |routes, view|
```

Comment:
It isn't obvious from the name of `bind_data` that it yields `routes` and `view`. I can't come up with a better name, but thought I'd throw it out and see if you had any ideas.